ibind/support/oauth.py

```diff
@@ -0,0 +1,359 @@
+import base64
+import configparser
+import random
+import string
+import time
+from typing import Optional
+from urllib import parse
+
+from Crypto.Cipher import PKCS1_v1_5 as PKCS1_v1_5_Cipher
+from Crypto.Hash import SHA256, HMAC, SHA1
+from Crypto.PublicKey import RSA
+from Crypto.Signature import PKCS1_v1_5 as PKCS1_v1_5_Signature
+from cryptography.hazmat.primitives.serialization import load_pem_parameters
+
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:  # pragma: no cover
+    from ibind import IbkrClient
+
+
+def req_live_session_token(client: 'IbkrClient') -> tuple[str, int]:
+    """ Get live session token and access token from IBKR Web API used to make API endpoint calls """
+
+    ENCRYPTION_METHOD = "RSA-SHA256"
+    REQUEST_URL = "oauth/live_session_token"
+    REQUEST_BASE_URL = "https://api.ibkr.com/v1/api/"
+    REQUEST_METHOD = "POST"
+
+    config = configparser.ConfigParser()
```

Comment:
Do you think we're at a stage when we'd want to turn this config reading into env vars as we've discussed?

Reply:
Yes can do, but do you have any good ideas on how to work with Paper and Live accounts?
That is one of the key benefits I've found of an env file, I can store all the variables separately and switch over easily.

Reply:
@Voyz I've replied to your comments in the file review section. I couldn't figure out how to copy the code and comment from file review to this conversation and gave up.

Reply:
You just provide different env vars for paper and live. There are many ways to set these when running, including doing so automatically from an env file. In my IDE I have different tasks that have either paper or live credentials. On cloud there are different instances deployed, each having their own credentials. Env files are just a different way to do this.

Reply:
Ok, I'll start adding variables from config to the system variables.
Please can you share a link or instructions on how to select the right account: "There are many ways to set these when running, including doing so automatically from an env file"

Reply:
Sorry, I'm not sure I'm understanding. How do you mean 'how to select the right account'? Do you mean how to provide these environment variables on runtime? That you can see here:

https://docs.docker.com/reference/cli/docker/container/run/#env
https://stackoverflow.com/questions/30494050/how-do-i-pass-environment-variables-to-docker-containers

Reply:
I have one set of variables for my paper account and another for my live account, they are stored in different env files and I can change the code to import which ever one I need.
When the variables are saved in the os environment variables, is there a simple way to do the same?

Reply:
Maybe create the option in ```var.py``` to use either an env file or the os environment variables?
```import configparser
from dotenv import load_dotenv
load_dotenv()
config = configparser.ConfigParser()
config.read('D:\\git_repos\\oauth_env\\oauth_test.env')
if os.environ.get('IBIND_LOG_TO_CONSOLE') is not None:
    LOG_TO_CONSOLE=os.environ.get('IBIND_LOG_TO_CONSOLE')
else:
    LOG_TO_CONSOLE=config['ibkr']['IBIND_LOG_TO_CONSOLE']```

Reply:
While I can see the potential flexibility and appreciate you suggesting the changes here, I'd prefer to avoid diverging into changing the configuration system as part of this PR. The focus here should remain on implementing OAuth support.

Changing how configuration is handled would affect more than just OAuth - it impacts the broader design and usage patterns of the library. If you'd like to continue this conversation, can I ask you to open a separate issue where we can discuss on the pros and cons of introducing config file support and how it might coexist with the current env var aproach? This way, we're open to input from others and ensure we fully evaluate the implications.

And as for this PR, let's keep it focused on introducing OAuth support 👍